KsApi/models/graphql/adapters/Backing+BackingFragment.swift

```diff
@@ -0,0 +1,82 @@
+import Foundation
+
+extension Backing {
```

Comment:
Maybe add a `TODO:// Remove file once all instances of this function are replaced by Backing+BackingFragment` in `Backing+GraphBacking`